include/seqan3/io/alignment_file/format_bam.hpp

```diff
@@ -490,7 +490,7 @@ class alignment_file_input_format_REMOVEME<format_bam> : alignment_file_input_fo
         return std::tuple{operations, ref_length, seq_length};
     }
 
-    // inherit read_field function from format_sam
+    //!\brief inherit read_field function from format_sam
```

Comment:
using declarations importing names are usually ignored  by doxygen. Did this change?

Reply:
I'm not sure. I even don't know why changing something from protected to public causes the using to be interpreted differently. :confused:

Reply:
This will all be changed in PR #1293 anyway and then I'' make sure all documentation is as desired. But I would like to keep the changes here low so I don't have to many rebasing issues.